ui/imports/shared/views/ExistingContacts.qml

```diff
@@ -38,20 +41,32 @@ Item {
         id: contactListView
         anchors.fill: parent
         rightMargin: 0
-        spacing: 0
+        leftMargin: 0
+        spacing: Style.current.padding
 
         model: root.contactsStore.myContactsModel
-        delegate: Contact {
+        delegate: StatusMemberListItem {
             width: contactListView.availableWidth
-            showCheckbox: root.showCheckbox
-            isChecked: root.pubKeys.indexOf(model.pubKey) > -1
             pubKey: model.pubKey
```

Comment:
```suggestion
            pubKey: Utils.getCompressedPk(model.pubKey)
```

Reply:
So Design is wrong about visual format of pubKey ?

Reply:
Yes, we should use compressed pubkey form everywhere.